code/controllers/subsystem/blackbox.dm

```diff
@@ -270,6 +270,25 @@ Versioning
 	key = new_key
 	key_type = new_key_type
 
+/datum/controller/subsystem/blackbox/proc/LogAhelp(ticket, action, message, recipient, sender)
+
+	if(!SSdbcore.Connect())
+		return
+
+	ticket = sanitizeSQL(ticket)
+	action = sanitizeSQL(action)
+	message = sanitizeSQL(message)
+	recipient = sanitizeSQL(recipient)
+	sender = sanitizeSQL(sender)

```

Comment:
To handle nullable values replace sender and recipient with the like of
```
sender = sender ? "'sanitizeSQL(sender)'" : "NULL"
```
And remove the apostrophes from around them in the SQL string as otherwise that'd insert the word null instead of value.